libs/api/domains/assets/src/lib/api-domains-assets.resolver.ts

```diff
@@ -0,0 +1,75 @@
+import { UseGuards } from '@nestjs/common'
+import { Query, Args } from '@nestjs/graphql'
+import graphqlTypeJson from 'graphql-type-json'
+import type { User } from '@island.is/auth-nest-tools'
+import {
+  CurrentUser,
+  IdsAuthGuard,
+  IdsUserGuard,
+  ScopesGuard,
+} from '@island.is/auth-nest-tools'
+import { Audit } from '@island.is/nest/audit'
+import {
+  GetRealEstateInput,
+  GetPagingTypes,
+  GetMultiPropertyInput,
+} from '../dto/getRealEstateInput.input'
+import { PropertyOwnersModel } from '../models/propertyOwners.model'
+import { UnitsOfUseModel } from '../models/propertyUnitsOfUse.model'
+import { AssetsXRoadService } from './api-domains-assets.service'
+
+@UseGuards(IdsAuthGuard, IdsUserGuard, ScopesGuard)
+@Audit({ namespace: '@island.is/api/assets' })
+export class AssetsXRoadResolver {
+  constructor(private assetsXRoadService: AssetsXRoadService) {}
+
+  @Query(() => graphqlTypeJson)
+  @Audit()
+  async getRealEstates(
```

Comment:
The GraphQL names are not following the island.is naming conventions:
https://docs.devland.is/technical-overview/api-design-guide/graphql-naming-conventions

Reply:
queries/types/inputs etc.